content/ko/docs/concepts/workloads/controllers/replicaset.md

```diff
@@ -37,7 +37,7 @@ OwnerReference가 없거나, OwnerReference가 {{< glossary_tooltip term_id="con
 레플리카셋은 지정된 수의 파드 레플리카가 항상 실행되도록 보장한다.
 그러나 디플로이먼트는 레플리카셋을 관리하고 다른 유용한 기능과 함께
 파드에 대한 선언적 업데이트를 제공하는 상위 개념이다.
-따라서 우리는 사용자 지정 오케스트레이션이 필요하거나 업데이트가 전혀 필요하지 않은 경우라면
+따라서 우리는 사용자 지정 오케스트레이션이 필요하거나 업데이트가 전혀 필요하지 않은 경우가 아니라면
 레플리카셋을 직접적으로 사용하기 보다는 디플로이먼트를 사용하는 것을 권장한다.
```

Comment:
```
Therefore, we recommend using Deployments instead of directly using ReplicaSets, unless
you require custom update orchestration or don't require updates at all.
```

```suggestion
따라서 별도의 사용자 정의(custom) 오케스트레이션이 필요하거나 업데이트가 전혀 필요 없는 경우가 아니라면,
레플리카셋을 직접 사용하기 보다는 디플로이먼트를 사용하는 것을 권장한다.
```

오류 정정 감사합니다! :)
추가적으로 해당 문장이 다소 길어서, 조금이나마 간략하게 개선해보았습니다. 참고하셔서 필요시 업데이트를 부탁드립니다.

Reply:
넵 새로 오픈하겠습니다~